File: web/src/main/java/top/bettercode/summer/web/error/IErrorRespEntityHandler.java
package top.bettercode.summer.web.error;

import org.springframework.web.context.request.RequestAttributes;
import top.bettercode.summer.web.IRespEntity;
import top.bettercode.summer.web.RespEntity;

/**
 * @author Peter Wu
 */
public interface IErrorRespEntityHandler {

  IRespEntity handle(RequestAttributes requestAttributes, RespEntity<Object> respEntity);

}

File: web/src/main/kotlin/top/bettercode/summer/web/error/IErrorRespEntityHandler.kt
package top.bettercode.summer.web.error

import org.springframework.web.context.request.RequestAttributes
import top.bettercode.summer.web.IRespEntity
import top.bettercode.summer.web.RespEntity

/**
 * @author Peter Wu
 */
interface IErrorRespEntityHandler {
    fun handle(requestAttributes: RequestAttributes?, respEntity: RespEntity<Any>?): IRespEntity
}
